components/ThreeDotsMenü/styles.js

```diff
@@ -0,0 +1,46 @@
+import styled from "styled-components";
+
+export const StyledLink = styled.a`
```

Comment:
Is there a reason why this is a normal anchor tag instead of the `Link` component?